### ISSUE TITLE
Where are the codes of dra fine-tuning of the model ?

### ISSUE THREAD

**Issue by 1024-program** [alibaba/easyrobust#7]
In the paper Towards Understanding and Boosting Adversarial Transferability from a Distribution Perspective,fine-tuning the model for the DCG loss in order to simulate data distribution. But I cannot find the codes of the fine-tuning, please could you release it or tell me where it is?

**Comment by ZY123-GOOD** [comment]
Thanks for your interest. This is a similar issue to https://github.com/alibaba/easyrobust/issues/5.
I appreciate your interest in our work. Due to ethical safety, we have not shared our codes for training to avoid malicious usage from attackers.
Below, we found an interesting implementation of our paper in GitHub. Thanks to the authors of this project.
https://github.com/JerryMazeyu/DRA-BlackBoxAttack
